HoloPipelines/pipelines/pipelineDicom2glb.py

```diff
@@ -2,17 +2,21 @@
 from components import compDicom2numpy
 from components import compNumpy2obj
 from components import compObj2glbWrapper
+from components import compNumpyTransformation
 import pathlib
 import sys
 import logging
+import datetime
 
 logging.basicConfig(level=logging.INFO)
 
 
 def main(dicomFolderPath, outputGlbPath, threshold):
     generatedNumpyList = compDicom2numpy.main(str(pathlib.Path(dicomFolderPath)))
+    resizedNumpyList = compNumpyTransformation.sizeLimit(generatedNumpyList, limit=256)
```

Comment:
if 256 is limit for all pipelines, or at least the default, maybe we can bring it to the shared configuration file?

Reply:
How did you decide for 256 for all others, and why does the abdomen pipeline have a different value?

Reply:
I added a `config.ini` in the base level of HoloPipeline. At the moment it only holds default value for resolution limit. This value is called as default value in `compNumpyTransformation`

Resolution limit of abdomen pipeline is now set to default (256). I chose 256 for default as the final outputs from experiment did not show significant downgrade in quality, also that this value is approximately the median to the values I usually get from most Dicom after resampling (220 to 300). We also plan to implement fast quadratic simplification (wrapper component still in my local branch) we can expect the file to be smaller.